packages/markdown/components/Image/index.jsx

```diff
@@ -1,18 +1,52 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
I think in the long run we should pull this kind of super UX-y type of behavior in to the UI. (Which is to say so I'm not too worried about the long-term accessibility concerns of this implementation.)

Reply:
Yeah we should definitely have a `<Lightbox>` component in the UI that we could use here instead.